turbo.json

```diff
@@ -3,7 +3,7 @@
   "pipeline": {
     "build": {
       "dependsOn": ["^build"],
-      "outputs": ["dist/**"]
+      "outputs": [".next/**", "!.next/cache/**", "dist/**"]
```

Comment:
This change errors with:

```
ENOENT: no such file or directory, lstat '/vercel/path0/node_modules/.pnpm/next@13.4.3_@babel+core@7.21.0_react-dom@18.2.0_react@18.2.0/node_modules/next/dist/server/next-server.js'
```

It happens on the first deployment of every PR, we had a lot of issues with this. We have no idea how to fix it though (we asked around).

cc @cramforce